mopidy_soundcloud/soundcloud.py

```diff
@@ -382,36 +370,75 @@ def parse_track(self, data, remote_url=False):
         return Track(**track_kwargs)
 
     def _update_public_client_id(self):
-        self.public_client_id = get_client_id()
+        """ Gets a client id which can be used to stream publicly available tracks """
+
+        def get_page(url):
+            return urlopen(url).read().decode("utf-8")
```

Comment:
Can we please do these requests using the user's configured proxy configuration.

Reply:
My point of moving it in here was to use `self.public_stream_client` to get the page

Reply:
It is possible, however it takes 25 seconds to load in my case

Reply:
I don't see 25 seconds but I do get a nasty delay. We can't ignore the user's proxy config though.

Reply:
It's the later script requests that seem to be slow:
```
SoundCloudBackend-11 ERROR    2021-04-06 17:36:53,337 fetch https://soundcloud.com/
SoundCloudBackend-11 ERROR    2021-04-06 17:36:53,463 fetch https://a-v2.sndcdn.com/assets/14-d8c9d717.js
SoundCloudBackend-11 ERROR    2021-04-06 17:36:54,059 fetch https://a-v2.sndcdn.com/assets/49-7141e628.js
SoundCloudBackend-11 ERROR    2021-04-06 17:36:54,110 fetch https://a-v2.sndcdn.com/assets/1-f2928cd3.js
SoundCloudBackend-11 ERROR    2021-04-06 17:36:58,222 fetch https://a-v2.sndcdn.com/assets/3-a24bc1da.js
SoundCloudBackend-11 ERROR    2021-04-06 17:37:01,144 Updated SoundCloud public client id to: PMINl7wzKHg40zDFBjAeIgMPWO4Dpw8M
```